conan/api/subapi/lockfile.py

```diff
@@ -27,8 +27,8 @@ def get_lockfile(self, lockfile=None, conanfile_path=None, cwd=None, partial=Fal
         :param conanfile_path: The full path to the conanfile, if existing
         :param lockfile: the name of the lockfile file
         """
```

Comment:
Please, review this docstring because it's saying:
```
If lockfile is explicitly defined, it would be either absolute or relative to cwd
the lockfile file must exist. If lockfile="None" (as string, no lockfile will be used)
```

Reply:
Sorry, my bad, not committing changes. Fixed.